Makefile

```diff
@@ -24,7 +24,7 @@ endif
 
 ROOT_DIR        := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
 PKG             := github.com/oracle/mysql-operator
-REGISTRY        := wcr.io/oracle
+REGISTRY        := iad.ocir.io/oracle
```

Comment:
Might want to change this to iad.ocir.io/spinnaker to make sure local make pushes go to our tenancy and not oracle which we'll preserve for tagged releases only.